dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/search/SearchServiceBean.java

```diff
@@ -364,7 +377,10 @@ public SolrQueryResponse search(DataverseRequest dataverseRequest, List<Datavers
         String titleSolrField = null;
         try {
             DatasetFieldType titleDatasetField = datasetFieldService.findByName(DatasetFieldConstant.title);
-            titleSolrField = titleDatasetField.getSolrField().getNameSearchable();
+            titleSolrField = solrFieldFactory.getSolrField(titleDatasetField.getName(),
+                                                           titleDatasetField.getFieldType(),
+                                                           titleDatasetField.isThisOrParentAllowsMultipleValues(),
+                                                           titleDatasetField.isFacetable()).getNameSearchable();
```

Comment:
Maybe it seems silly, but with formatting like that it's not obvious that we are performing .getNameSearchable(). 
I propose to use stream-like code formatting and put .getNameSearchable() in a new line or to stay consistent with other uses of SolrField dsfSolrField = solrFieldFactory.getSolrField to firstly assign this and then the final variable.